test/evergreen.yml

```diff
@@ -2805,14 +2806,16 @@ tasks:
         vars:
           # CC is set to the system default "clang" binary here as a workaround. Change it back
           # to mongodbtoolchain "clang" binary after BUILD-10248 fix.
-          configure_env_vars:
-            CCAS=/opt/mongodbtoolchain/v3/bin/gcc CC=/usr/bin/clang
-            CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH
-            CFLAGS="-ggdb -fPIC -fsanitize=address -fno-omit-frame-pointer
-            -I/opt/mongodbtoolchain/v3/lib/gcc/ppc64le-mongodb-linux/8.2.0/include"
-            CXXFLAGS="-ggdb -fPIC -fsanitize=address -fno-omit-frame-pointer
-            -I/opt/mongodbtoolchain/v3/lib/gcc/ppc64le-mongodb-linux/8.2.0/include"
-          posix_configure_flags: --enable-diagnostic --with-builtins=lz4,snappy,zlib
+          posix_configure_flags:
+            -DHAVE_DIAGNOSTIC=1
```

Comment:
Can we add `-DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/clang.cmake` as the first line here too? For consistency.

Reply:
f4cc43a3ecb8eddd9d439644b0a49b5f7ef8ddaa